### DISCUSSION TITLE
Logic analyzer not working on pi pico

### DISCUSSION THREAD

**Discussion by Lup1n-3** [fhdm-dev/scoppy#88] [Q&A]
I paid for the premium version but the logic analyzer doesn't work, when I don't connect the signal cables it gives me a square wave but when I put gpio 6 or any other in a pwm signal for example,The wave is canceled and does not give a signal, I have everything well soldered, even in the gpio pins I put a 100k resistor

**Comment by fhdm-dev** [comment]
Hi,

It's hard to know what's going wrong without seeing your schematic or the settings in the app so please try the following:

- Ensure Logic analyzer mode is enabled in the app - Tap Menu and check the 'Mode'
- Connect GPIO22 directly to GPIO6 (without anything else connected to GPIO6). Do you see a square wave when Time/Div is set to 1ms/div? Is you do then I think there's a problem with your circuit. I don't know how you're using the 100k resistor but it may be causing the high logic level voltage at GPIO6 to drop below the required 2V (see table 5.3.3.4 in the RP2040 datasheet).
